Automattic/jetpack
# Jetpack | Update recommendations section links for Social

After the updated connections management UI, the Jetpack recommendation section still points to calypso for manage connections links.

## Proposed changes:

* Update Manage Connections link to the new connections management UI instead of Claypso

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Turn ON feature flag - `define( 'JETPACK_SOCIAL_USE_ADMIN_UI_V1', true );`
* Go to Jetpack > Dashboard > Recommendations - `/wp-admin/admin.php?page=jetpack#/recommendations/summary`
* Confirm that the Manage Connections link for Social points to Jetpack Sharing settings section
* Click on "Social Media Sharing" link to end up at `/wp-admin/admin.php?page=jetpack#/recommendations/publicize`
* Confirm that the Manage Connections link for Social points to Jetpack Sharing settings section
* Turn OFF feature flag
* Confirm that the above links point to Calypso

<table>
    <tbody>
        <tr>
            <th>BEFORE</th>
            <th>AFTER</th>
        </tr>
        <tr>
            <th colspan="2">All recommendations section</th>
        </tr>
        <tr>
            <td>
<img width="1715" alt="Screenshot 2024-06-03 at 3 18 50 PM" src="https://github.com/Automattic/jetpack/assets/18226415/3ea76a0f-2d59-47c3-bac0-46b9afb8f2bb">
</td>
            <td>
<img width="1716" alt="Screenshot 2024-06-03 at 3 16 15 PM" src="https://github.com/Automattic/jetpack/assets/18226415/565dfd0c-d201-4985-9c26-73493dea4e29">
</td>
        </tr>
        <tr>
            <th colspan="2">Publicize recommendations section</th>
        </tr>
        <tr>
            <td><img width="1717" alt="Screenshot 2024-06-03 at 3 19 30 PM" src="https://github.com/Automattic/jetpack/assets/18226415/e4c5765f-e6ad-473c-9cf9-11fffc55995b"></td>
            <td>
<img width="1715" alt="Screenshot 2024-06-03 at 3 16 43 PM" src="https://github.com/Automattic/jetpack/assets/18226415/68d945d7-7376-40f9-92c2-df7d04d4865a">
</td>
        </tr>
    </tbody>
</table>
